src/commcare_cloud/ansible/roles/nginx/tasks/install.yml

```diff
@@ -90,10 +91,33 @@
     owner: root
     group: root
   no_log: true
-  when: not fake_ssl_cert and nginx_key_value
+  when: not fake_ssl_cert and nginx_key_value is defined and nginx_key_value != ''
   tags:
     - update-cert
 
+
+- name: Link CCHQ SSL cert
+  become: yes
+  file:
+    src: "/etc/letsencrypt/live/{{SITE_HOST}}/fullchain.pem"
+    dest: "{{ ssl_certs_dir }}/{{ nginx_ssl_cert }}"
+    state: link
+  when: not fake_ssl_cert and letsencrypt_cchq_ssl is defined
+  tags:
+    - update-cert
+    - letsencrypt
+
+- name: Link CCHQ SSL Key
+  become: yes
+  file:
+    src: "/etc/letsencrypt/live/{{SITE_HOST}}/privkey.pem"
+    dest: "{{ ssl_keys_dir }}/{{ nginx_ssl_key }}"
+    state: link
+  when: not fake_ssl_cert and letsencrypt_cchq_ssl is defined
```

Comment:
Can just do ` and letsencrypt_cchq_ssl|default(False)`: https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#defaulting-undefined-variables